plio/tests.py

```diff
@@ -943,16 +950,25 @@ def test_question_metrics_answers_provided(self):
 
         # seed questions
         Question.objects.create(
-            type="mcq", item=item_1, text="test", options=["", ""], correct_answer=0
+            type="mcq",
+            item=item_1,
+            text="test",
+            options=["", ""],
+            correct_answer=0,
         )
         Question.objects.create(
             type="checkbox",
             item=item_2,
             text="test",
             options=["", ""],
             correct_answer=[0, 1],
+            survey=False,
```

Comment:
by default survey mode is `False` right? so why write here explicitly? the tests would run fine even if you didn't add this here right?